README.md

```diff
@@ -1295,7 +1465,7 @@ $ curl -v "localhost:8000/api/workflows/v1/validate" -F wdlSource=@src/main/reso
 HTTPie:
 
 ```
-$ http --print=hbHB --form POST localhost:8000/api/workflows/v1 wdlSource=@src/main/resources/3step.wdl workflowInputs@inputs.json
+$ http --print=hbHB --form POST localhost:8000/api/workflows/v1/validate wdlSource=@src/main/resources/3step.wdl workflowInputs@inputs.json
```

Comment:
This is in the "/validate" documentation section. Scroll up to the curl doc just above. I found this missing-"/validate"-in-the-url typo when doing a search and replace for the regular wdl submit url.